src/javis_viewer.jl

```diff
@@ -0,0 +1,146 @@
+# TODO: Intelligent way of handling slider
+
+#=
+FIXME:
+Fascinating bug I found.
+Apparently to create a list of actions anywhere, you must first define a Video object.
+Else, you get an error like this:
+
+```
+ERROR: LoadError: BoundsError: attempt to access 0-element Array{Video,1} at index [1]
+Stacktrace:
+ [1] getindex at ./array.jl:809 [inlined]
+ [2] Action(::UnitRange{Int64}, ::Nothing, ::Function; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/src/Projects/javis/src/Javis.jl:397
+ [3] Action at /home/src/Projects/javis/src/Javis.jl:397 [inlined]
+ [4] #Action#3 at /home/src/Projects/javis/src/Javis.jl:350 [inlined]
+ [5] Action(::UnitRange{Int64}, ::Function) at /home/src/Projects/javis/src/Javis.jl:350
+ [6] top-level scope at /home/src/Projects/javis/src/javis_viewer.jl:34
+ [7] include(::String) at ./client.jl:457
+ [8] top-level scope at REPL[1]:1
+in expression starting at /home/src/Projects/javis/src/javis_viewer.jl:34
+```
+
+=#
+
+function javis_viewer(; video::Video, frames::Int, action_list::Vector)
+
+    #####################################################################
+    # VIEWER WINDOW AND CONFIGURATION
+    #####################################################################
+
+    frame_dims = size(get_javis_frame(video, action_list, 1))
+    win = GtkWindow("Javis Viewer", frame_dims[1], frame_dims[2])
+
+    set_gtk_property!(win, :title, "Javis Viewer") # Sets title of window
+    set_gtk_property!(win, :border_width, 20) # Sets border size of window
+
+    #####################################################################
+    # DISPLAY WIDGETS
+    #####################################################################
+
+    slide = slider(1:frames) # Creates a slider
+    tbox = GtkReactive.textbox(Int; signal = signal(slide)) # Creates a textbox
+
+    forward = GtkButton("==>") # Button for going forward through animation
+    backward = GtkButton("<==") # Button for going backward through animation
```

Comment:
Not the best position for this comment but before I forget:
Can we make this to work on the left and right arrow button as I hate clicking :laughing:

Reply:
Got this fixed! :smile: